include/openssl/core_names.h

```diff
@@ -84,6 +88,12 @@ extern "C" {
 #define OSSL_MAC_PARAM_DIGESTSIZE   "digestsize" /* size_t */
 #define OSSL_MAC_PARAM_OUTLEN       "outlen"    /* size_t */
 
+/* Known MAC names (not a complete list) */
+#define OSSL_MAC_NAME_CMAC          "CMAC"
```

Comment:
Should doco get updated?

Reply:
Not sure why...  these names are mostly for our internal use, I'm not even sure why we make them public, other than to be nice...  It's not like we want to force anyone to use those macros.